x-pack/plugins/uptime/public/components/monitor/ping_list/ping_list.tsx

```diff
@@ -40,7 +40,7 @@ const ONE_SECOND_AS_MICROS = 1000000;
 // the limit for converting to seconds is >= 1 sec
 const MILLIS_LIMIT = ONE_SECOND_AS_MICROS * 1;
 
-export const formatDuration = (durationMicros: number) => {
+export const formatDuration = (durationMicros: number, shortLabel?: boolean) => {
```

Comment:
Does it make sense to move this helper function to a higher level outside of a specific component? I feel like we are going to encounter this duration formatting need all over.